<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="LoadData">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>86</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Train">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>86</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Train</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Recall">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>86</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Recall</string>
    </property>
   </widget>
   <widget class="QListView" name="DataLOG">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plotterRaw" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>501</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plotterTrainned" native="true">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>200</y>
      <width>491</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plotMSE" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>400</y>
      <width>211</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plotAccuracy" native="true">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>400</y>
      <width>211</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Hidden Layer</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1021</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
